e2e-tests/resources/ollama_test_config.yaml

```diff
@@ -0,0 +1,46 @@
+langextract:
+  prompt_file: langextract_prompts/default_pii_phi_prompt.txt
+  examples_file: langextract_prompts/default_pii_phi_examples.yaml
+  
+  entity_mappings:
+    person: PERSON
+    full_name: PERSON
+    name_first: PERSON
+    name_last: PERSON
+    name_middle: PERSON
+    location: LOCATION
+    address: LOCATION
+    organization: ORGANIZATION
+    phone: PHONE_NUMBER
+    phone_number: PHONE_NUMBER
+    email: EMAIL_ADDRESS
+    date: DATE_TIME
+    ssn: US_SSN
+    identification_number: US_SSN
+    credit_card: CREDIT_CARD
+    medical_record: MEDICAL_LICENSE
+    ip_address: IP_ADDRESS
+    url: URL
+    iban: IBAN_CODE
+    payment_status: IGNORE
```

Comment:
Why do we need to explicitly ignore entities? Do we get limited generalizability from LangExtract?